bin/peggy

```diff
@@ -33,8 +33,6 @@ function printHelp() {
   console.log("      --format <format>              format of the generated parser: amd,");
   console.log("                                     commonjs, globals, umd (default: commonjs)");
   console.log("  -h, --help                         print help and exit");
-  console.log("  -O, --optimize <goal>              select optimization for speed or size");
```

Comment:
I have a rewrite of the CLI that I'm still tweaking, so let's double-check that I get the same results there that you have here: `--optimize` is hidden from the help text, and causes a warning on stderr if used.

Reply:
Yes, all correct